brought up to date with current master (including normalization stuff)

BrannonKing opened the pull request:
All the unit tests run for me locally. I haven't yet tested the reproducible builds (or Windows build or OSX build). I'll do that next week.

I was thinking that you had already renamed bitcoind to be lbrycrdd. I guess I was up in the night on that. I haven't done it yet either.

lbrynaut commented:
I haven't done any work on this repo, the latest I had is here (aside from my current unpushed work):

https://github.com/lbryio/lbrycrd/pull/257

BrannonKing committed:
Rebase lbry on to upstream (WIP)

brought in the claimtrie tests, made it all compile

yanked the ICU stuff as it will come with the normalization branch

Add base58_keys_valid, Checkpoints tests, fix blockchain test

Signed-off-by: Anthony Fieroni <bvbfan@abv.bg>

Port tests, still has failing

Signed-off-by: Anthony Fieroni <bvbfan@abv.bg>

Claim trie, miner, coins tests fix

Signed-off-by: Anthony Fieroni <bvbfan@abv.bg>

Fix tx validation

Signed-off-by: Anthony Fieroni <bvbfan@abv.bg>

fixed nameclaim link


Fix transaction tests by removing not invalid tx

Signed-off-by: Anthony Fieroni <bvbfan@abv.bg>

fix overflow in amount_tests


Fix locking logic, miner tests pass

Signed-off-by: Anthony Fieroni <bvbfan@abv.bg>

Fix wallet tests

Signed-off-by: Anthony Fieroni <bvbfan@abv.bg>

Fix script and validation block tests

Signed-off-by: Anthony Fieroni <bvbfan@abv.bg>

Fix denial of service tests

Signed-off-by: Anthony Fieroni <bvbfan@abv.bg>

Continue rebasing LBRY on to bitcoin upstream.

brought up to date with current master


allow clean


better use of depends


don't use pkg-config


made the darwin cross-compile run


first pass at docker builds
